CHANGES.md

```diff
@@ -31,6 +31,11 @@
 
 ## Changes
 
+- Changes `JUPYTERHUB_VERSION` from `1.4.0-20210506` to `4.0.2-20230816`
+
+- Add an image to the list of images that can be launched from JupyterHub which will be used to start an instance of MLflow.
```

Comment:
This changelog is funny.  I think it has not been merged with latest of `master`.

Reply:
The merge was done before the version bump on master. I'll move the change to the unreleased section.